src/protocols/secure_channel/PASESession.cpp

```diff
@@ -299,7 +299,7 @@ CHIP_ERROR PASESession::Pair(const Transport::PeerAddress peerAddress, uint32_t
     CHIP_ERROR err = Init(myKeyId, peerSetUpPINCode, delegate);
     SuccessOrExit(err);
 
-    mExchangeCtxt = exchangeCtxt->Retain();
+    mExchangeCtxt = exchangeCtxt;
```

Comment:
The lifetime of the exchange context doesn't seem to be expressed via Retain() and Release(). The entity (PASESession in this scenario) is given the object, and it owns it for the lifetime of it. At the end, it's expected to call `Close()` on it.

I suspect this is done since CRMP also interacts with Exchange Contexts.

Reply:
Added the documentation.